tests/integration/extenders/SettingsTest.php

```diff
@@ -163,6 +163,38 @@ public function custom_setting_default_passed_to_callback()
         $this->assertArrayHasKey('customPrefix.noCustomSetting', $payload['data']['attributes']);
         $this->assertEquals('customDefaultModified2', $payload['data']['attributes']['customPrefix.noCustomSetting']);
     }
+
+    /**
+     * @test
+     */
+    public function custom_less_var_does_not_work_by_default()
+    {
+        $this->extend(
+            (new Extend\Frontend('forum'))
+                ->css(__DIR__.'/../../fixtures/less/config.less'),
+        );
+
+        $response = $this->send($this->request('GET', '/'));
+
+        $this->assertEquals(500, $response->getStatusCode());
+    }
+
+    /**
+     * @test
+     */
+    public function custom_less_var_works_if_registered()
+    {
+        $this->extend(
+            (new Extend\Frontend('forum'))
+                ->css(__DIR__.'/../../fixtures/less/config.less'),
+            (new Extend\Settings())
+                ->registerLessConfigVar('custom-config-setting', 'custom-prefix.custom_setting')
+        );
+
+        $response = $this->send($this->request('GET', '/'));
```

Comment:
How tricky would it be to actually check that the value is correctly declared in the compiled CSS assets?

```
$response = $this->send($this->request('GET', '/assets/forum.css'));

// I don't know the syntax at all...
$this->assertTrue(true, strpos($response->getData(), '--custom-config-setting:customValue;') === false
```

Reply:
For some reason the request to the file didn't work, though `forum.css` is no small file, so it could be memory consuming.

If the variable doesn't actually work, the LESS parser will error out and produce a 500 error, that should be enough for this test.

Reply:
> For some reason the request to the file didn't work, though forum.css is no small file, so it could be memory consuming.

It shouldn't, if you're trying to get it via a request, since static asset handling is done at the web server level.  Could we directly read it from the file system?